Allow connection from qfield.cloud in pg_hba.conf (#242)
opengisch/QField-docs: documentation/reference/qfieldcloud/specs.md

| app.qfield.cloud | 185.203.114.168 |

## PostgreSQL configuration

If your project contains PostgreSQL (PostGIS) layers, you need to configure your PostgreSQL server so it allows connection from `app.qfield.cloud` IP address.

Otherwise, you will get an error like this:

```

> FATAL: no pg_hba.conf entry for host "185.203.114.168", user "root", database "mydb_test", SSL off

Reviewer: ```suggestion
FATAL: no pg_hba.conf entry for host "185.203.114.168", user "qfc", database "mydb_test", SSL off
```

Let's not suggest a username like root is good to use

WARNING:QGIS_MSGLOG:1 unavailable layer(s) found:
```

> To do so, you need to edit the `pg_hba.conf` file where your PostgreSQL server is hosted. For more information, check other people experience from [StackOverflow questions](https://stackoverflow.com/search?q=FATAL+no+pg_hba.conf+entry+for+host).

Reviewer: ```suggestion
To do so, you need to edit the `pg_hba.conf` file where your PostgreSQL server is hosted. For more information, refer to recommendations on [StackOverflow questions](https://stackoverflow.com/search?q=FATAL+no+pg_hba.conf+entry+for+host).
```
